frontend/packages/dev-console/src/components/helm/HelmReleaseHeader.tsx

```diff
@@ -0,0 +1,53 @@
+import { sortable } from '@patternfly/react-table';
+
+export const tableColumnClasses = [
```

Comment:
Any reason to keep this an array with comments for naming?  Why not:

```typescript
export const tableColumnClasses = {
   name: '',
   revision: '',
   timestamp: ''
   // etc.
}
```
Make the use obvious to the user, eliminate the need to `index[x]`.

Reply:
Done.